main/core/src/mill/define/Task.scala

```diff
@@ -40,22 +40,74 @@ abstract class Task[+T] extends Task.Ops[T] with Applyable[Task, T] {
   def self: Task[T] = this
 }
 
+object Task {
```

Comment:
This is just moved up from the bottom of the file, so it is next to the companion `trait`